jobsets/default.nix

```diff
@@ -1,4 +1,4 @@
-{ nixopsPrsJSON, cardanoPrsJSON, nixpkgs ? <nixpkgs>, declInput ? {} }:
+{ nixopsPrsJSON ? ./simple-pr-dummy.json, cardanoPrsJSON ? ./simple-pr-dummy.json, nixpkgs ? <nixpkgs>, declInput ? {} }:
```

Comment:
they make testing outside of hydra far simpler, you can just nix-build the file and get json you can diff